app/models/prisoner.rb

```diff
@@ -8,4 +8,14 @@ class Prisoner < ActiveRecord::Base
   validates :surname, presence: true
   validates :date_of_birth, presence: true
   validates :gender, presence: true
+
+  def self.search(query=[])
+    return [] if query.nil?
```

Comment:
@dan-palmer no it's fine, you're right it should be chainable, especially given I'm giving the param a default value. This PR isn't actually ready to merge yet, a few more bits to do on it. But you're welcome to keep reviewing and piping in with comments.